src/app/components/Notification/Notification.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import AppConfigStore from '../../stores/AppConfigStore';
+
+const Notification = (props) => {
+  const appConfigState = AppConfigStore.getState();
+  const {
+    notificationType,
+  } = props;
+
+  const notification = appConfigState[notificationType];
+
+  if (!notification) return null;
+
+  return (
+    <div className="nypl-banner-alert">
+      <p style={{ padding: '10px 20px 0px', margin: 0 }} dangerouslySetInnerHTML={{__html: notification }} />
```

Comment:
So minor, but if it where `padding: '0px 20px 0px'` the message will be vertically centered instead of slightly lower. I had that change in the PR that was closed.

Reply:
Fixed